ambry-commons/src/test/java/com.github.ambry.commons/BlobIdV2.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.commons;
+
+import com.github.ambry.clustermap.ClusterMap;
+import com.github.ambry.clustermap.PartitionId;
+import java.io.IOException;
+
+
+/**
+ *  A class that makes {@link BlobId#getCurrentVersion()} to return {@link BlobId#BLOB_ID_V2}, which will serialize
+ *  a blobId to {@link BlobIdV2}.
+ */
+public class BlobIdV2 extends BlobId {
```

Comment:
This is for testing the case when get/head/delete blobIds in v2 with meaningful aid and cid. The current blobId does not enable the write path, so not easy to make blobId string in v2.